.github/workflows/publish.sh

```diff
@@ -4,21 +4,28 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# from https://cloud.google.com/sdk/docs/downloads-apt-get
-export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)"
-echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
-curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-sudo apt-get update && sudo apt-get install google-cloud-sdk
-
 gcloud config set disable_prompts True
 gcloud auth activate-service-account --key-file <(echo ${GCLOUD_CLIENT_SECRET} | base64 --decode)
 gcloud auth configure-docker
 
-readonly root=$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd)
 readonly version=$(cat ${root}/VERSION)
 readonly git_branch=${GITHUB_REF:11} # drop 'refs/head/' prefix
 readonly git_timestamp=$(TZ=UTC git show --quiet --date='format-local:%Y%m%d%H%M%S' --format="%cd")
 readonly slug=${version}-${git_timestamp}-${GITHUB_SHA:0:16}
 
-echo "Publishing riff nop provisioner"
-(cd $root && KO_DOCKER_REPO="gcr.io/projectriff/nop-provisioner" ko publish -t "${version}" -t "${slug}" github.com/projectriff/nop-provisioner/cmd/provisioner)
+publishImage() {
+  local tag=$1
+  local source=ko.local/provisioner-735f690b778af94ebf6fae1b426bbf1d:latest
```

Comment:
I believe you left out a hardcoded path used for testing here

Reply:
that's the image built by `ko publish -L github.com/projectriff/nop-provisioner/cmd/provisioner`

Reply:
But we don't want to edit this file everytime the code changes, do we?

Reply:
> But we don't want to edit this file everytime the code changes, do we?

It will only change if we rename the package. The value is a hash of the import path, not the content:

```
$ echo -n github.com/projectriff/nop-provisioner/cmd/provisioner | md5
735f690b778af94ebf6fae1b426bbf1d
```

Reply:
The things you learn on a Friday...